codespell_lib/_codespell.py

```diff
@@ -361,12 +362,45 @@ def parse_options(args):
                         help='print LINES of leading context')
     parser.add_argument('-C', '--context', type=int, metavar='LINES',
                         help='print LINES of surrounding context')
+    parser.add_argument('--config', type=str,
+                        help='path to config file.')
 
     parser.add_argument('files', nargs='*',
                         help='files or directories to check')
 
+    # Parse command line options.
     options = parser.parse_args(list(args))
 
+    # Load config files and look for ``tool:codespell`` options.
+    if (
+        options.config
+        or os.path.exists('setup.cfg')
+        or os.path.exists('.codespellrc')
+    ):
+        cfg_files = ['setup.cfg', '.codespellrc']
+        if options.config:
+            cfg_files.append(options.config)
+        config = configparser.ConfigParser()
+        config.read(cfg_files)
```

Comment:
Ohh this already handles a list.

Never mind the above. I guess this already works as it should, then?

Reply:
Yes, please read configparser documentation: https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read